### DISCUSSION TITLE
How to publish a plugin

### DISCUSSION THREAD

**Discussion by cou723** [Flow-Launcher/Flow.Launcher#2807] [Q&A] [answered]
https://github.com/cou723/Flow.Plugin.CursorWorkspace

I have created a plugin for Cursor Editor based on VSCodeWorkspace but do not know how to publish it.
How can I publish it to the plugin store?
Sorry if I have missed something in the documentation. If so, I would appreciate it if you could post a link to the documentation.

**Comment by onesounds** [comment, marked as answer]
🧙 https://github.com/Flow-Launcher/Flow.Launcher.PluginsManifest
